modules/similarity-algorithms/pages/cosine-similarity.adoc

```diff
@@ -0,0 +1,95 @@
+= Cosine Similarity
+:stem: latex
+:experimental:
+
+Algorithm link: link:https://github.com/tigergraph/gsql-graph-algorithms/tree/277349ce0414ba797edcad1488b6caf1904d5beb/algorithms/Similarity/approximate_nearest_neighbors[Approximate Nearest Neighbors]
```

Comment:
This is a UDF function, not an algorithm.  Note the location of the UDF file, specified in the updated DOC ticket.

Reply:
Changed this.